home-manager/homemade.nix

```diff
@@ -0,0 +1,80 @@
+{ pkgs, lib, ... }:
+
+# FAQ
+#
+# A. How to make executable? .text= makes syms, that links to non executable file
+# Q. https://github.com/nix-community/home-manager/blob/15043a65915bcc16ad207d65b202659e4988066b/modules/xsession.nix#L195C1-L197
+
+# - Tiny tools by me, they may be rewritten with another language.
+{
+  # - Keep *.bash in shellscript naming in this repo for maintainability, the extname should be trimmed in the symlinks
+  xdg.dataFile."homemade/bin/add_nix_channels".source = ../home/.local/share/homemade/bin/add_nix_channels.bash;
```

Comment:
Keep this for now, because I don't have strong confidence in this way yet. And this script is used in very bootstrapping phase...